src/components/LocaleSelector.vue

```diff
@@ -45,10 +46,11 @@ export default {
   },
   methods: {
     updateRouter({ target: { value: currentLocale } }) {
+      const slug = getSlug(currentLocale);
       this.$i18n.locale = currentLocale;
```

Comment:
You may want to consider re-using the `updateCurrentLocale` utility to avoid repeating the `$i18n` and `<meta>` logic here.